lib/bif.ml

```diff
@@ -36,4 +36,6 @@ let type_sigs = [
      Type.(of_elem (TyFun ([Type.bool; Type.bool], Type.bool))));
     ({module_name="erlang"; function_name="not"; arity=1},
      Type.(of_elem (TyFun ([Type.bool], Type.bool))));
+    ({module_name="erlang"; function_name="abs"; arity=1},
```

Comment:
`abs` is not in PLT, right?

Reply:
Right.
- https://github.com/dwango/fialyzer/issues/170
- https://github.com/erlang/otp/blob/OTP-21.1.4/lib/hipe/cerl/erl_bif_types.erl

Reply:
OK. Thank you for the links!

LGTM.